VideoAPI/VideoApi.Contract/Requests/ConferenceEventRequest.cs

```diff
@@ -48,5 +48,10 @@ public class ConferenceEventRequest
         /// Event reason
         /// </summary>
         public string Reason { get; set; }
+
+        /// <summary>
+        /// Can be either the phone number or "anonymous"
+        /// </summary>
+        public string Phone { get; set; }
```

Comment:
Do we handle Anonymous any differently to a phone number?

Reply:
I dont think so, nothing mentioned on the ticket